recipe/meta.yaml

```diff
@@ -29,10 +29,12 @@ requirements:
     # Update version number in URL before checking dependencies
     # https://github.com/python-poetry/poetry/blob/1.7.1/pyproject.toml
     - python >=3.8,<4.0
-    - poetry-core 1.8.1
+    - poetry-core 1.9.0
     - poetry-plugin-export >=1.6.0,<2.0.0
     - python-build >=1.0.3,<2.0.0
-    - cachecontrol-with-filecache >=0.13.0,<0.14.0
+    # the following syntax makes the update-grayskull bot happier
+    - cachecontrol >=0.14.0,<0.15.0
+    - cachecontrol-with-filecache
```

Comment:
@maresb, I've found in working with other conda-packages that are equivalent to pip extras that this syntax works better with the `inspection: update-grayskull` setting.  This way, we don't have to manually set the constraints on `cachecontrol-with-filecache` (which `grayskull` doesn't recognize) because they're pinned to match `cachecontrol`.  But the bot does automatically update the constraints on `cachecontrol` itself.

Reply:
😊